lib/course/middleware/logger.rb

```diff
@@ -0,0 +1,21 @@
+require "logger"
+
+module Course
+  module Middleware
+    class Logger
+      attr_reader :app, :logger
+
+      def initialize(app, logger:)
+        raise ArgumentError, "logger can not be nil" if logger.nil?
+
+        @app = app
+        @logger = logger
+      end
+
+      def call(env)
+        logger.info("#{env['REQUEST_METHOD']} #{env['REQUEST_URI']}")
+        app.call(env)
```

Comment:
You did the first part, but not the second of this comment:
https://github.com/XaoGao/rack-middleware/pull/4#discussion_r1125512251

Reply:
fixed, add to env hash logger obj